Index: src/main/java/org/mindswap/model/ClientToken.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.mindswap.model;\n\nimport jakarta.persistence.*;\nimport lombok.AllArgsConstructor;\nimport lombok.Builder;\nimport lombok.Data;\nimport lombok.NoArgsConstructor;\n\n@Data\n@Builder\n@NoArgsConstructor\n@AllArgsConstructor\n@Entity\npublic class ClientToken {\n\n    @Id\n    @GeneratedValue\n    public Integer id;\n\n    @Column(unique = true)\n    public String token;\n\n    @Enumerated(EnumType.STRING)\n    public TokenType tokenType = TokenType.BEARER;\n\n    public boolean revoked;\n\n    public boolean expired;\n\n    @ManyToOne\n    @JoinColumn(name = \"user_email\")\n    public String email;\n}
===================================================================
diff --git a/src/main/java/org/mindswap/model/ClientToken.java b/src/main/java/org/mindswap/model/ClientToken.java
--- a/src/main/java/org/mindswap/model/ClientToken.java	
+++ b/src/main/java/org/mindswap/model/ClientToken.java	
@@ -22,12 +22,11 @@
 
     @Enumerated(EnumType.STRING)
     public TokenType tokenType = TokenType.BEARER;
-
     public boolean revoked;
-
     public boolean expired;
 
     @ManyToOne
     @JoinColumn(name = "user_email")
     public String email;
+    public Role role;
 }
\ No newline at end of file
